com.unity.cinemachine/Editor/Windows/CinemachineCorePrefs.cs

```diff
@@ -0,0 +1,92 @@
+using UnityEngine;
+using UnityEditor;
+
+namespace Cinemachine.Editor
+{
+    static class CinemachineCorePrefs
+    {
+        static CinemachineSettings.BoolItem SettingsFoldedOut = new("CNMCN_Core_Folded", true);
+
+        public static CinemachineSettings.BoolItem StoryboardGlobalMute = new ("CNMCN_StoryboardMute_Enabled", false);
+        public static CinemachineSettings.BoolItem DraggableComposerGuides = new ("CNMCN_DraggableComposerGuides_Enabled", true);
+        public static CinemachineSettings.BoolItem ShowBrainIconInHierarchy = new ("CNMCN_Core_Brain_icon_in_hierarchy", true);
+        public static CinemachineSettings.BoolItem ShowInGameGuides = new ("CNMCN_Core_ShowInGameGuides", true);
+        public static CinemachineSettings.ColorItem ActiveGizmoColour = new ("CNMCN_Core_Active_Gizmo_Colour", new Color32(255, 0, 0, 100));
+        public static CinemachineSettings.ColorItem InactiveGizmoColour = new ("CNMCN_Core_Inactive_Gizmo_Colour", new Color32(9, 54, 87, 100));
+        public static CinemachineSettings.ColorItem BoundaryObjectGizmoColour = new ("CNMCN_Core_BoundaryObject_Gizmo_Colour", Color.yellow);
+
+        public static readonly GUIContent s_StoryboardGlobalMuteLabel = new GUIContent(
+            "Storyboard Global Mute", 
+            "If checked, all storyboards are globally muted.");
+        public static readonly GUIContent s_ShowInGameGuidesLabel = new GUIContent(
+            "Game View Guides", 
+            "Enable the display of overlays in the Game View.  You can adjust colours and opacity in Cinemachine Preferences.");
+        public static readonly GUIContent s_SaveDuringPlayLabel = new GUIContent(
```

Comment:
I am not sure about the naming convention here, because they are public.

Reply:
This is internal code that disappear when we move to UITK, so I don't really care